ingest/cell_metadata.py

```diff
@@ -141,21 +150,34 @@ def transform(self):
                 if not self.numeric_array_columns.get(annot_name)
                 else "group"
             )
-            yield AnnotationModel(
-                annot_header,
-                self.Model(
-                    {
-                        "name": annot_name,
-                        "annotation_type": stored_mongo_annot_type,
-                        # unique values from "group" type annotations else []
-                        "values": list(self.file[annot_header].unique())
-                        if annot_type == "group"
-                        else [],
-                        "study_file_id": self.study_file_id,
-                        "study_id": self.study_id,
-                    }
-                ),
-            )
+
+            group = True if annot_type == "group" else False
+            # should not store annotations with >200 unique values for viz
+            # annot_header is the column of data, which includes name and type
+            # large is any annotation with more than 200 + 2 unique values
+            large = True if len(list(self.file[annot_header].unique())) > 202 else False
+
+            if large and group:
+                skip_msg = f"\'{annot_name}\' not stored for visualization - too many unique values "
+                user_logger.error(skip_msg)
+                print(skip_msg)
+                continue
+            else:
+                yield AnnotationModel(
+                    annot_header,
+                    self.Model(
+                        {
+                            "name": annot_name,
+                            "annotation_type": stored_mongo_annot_type,
+                            # unique values from "group" type annotations else []
+                            "values": list(self.file[annot_header].unique())
+                            if annot_type == "group"
+                            else [],
```

Comment:
Reuse variables from above for readability, and to avoid the dreaded multi-line ternary.

```suggestion
                            "values": unique_values if group else [],
```

Reply:
Adjusting here to fit Jon's original conception - ingest all metadata, just don't store **values** for large group.